build/musicbrainz-dev/Dockerfile

```diff
@@ -67,6 +70,22 @@ RUN echo "deb [signed-by=/etc/apt/keyrings/nodesource.asc] https://deb.nodesourc
         && \
     rm -rf /var/lib/apt/lists/* && \
     cd /usr/src && \
+    curl -sSLO https://cpan.metacpan.org/authors/id/P/PE/PEVANS/perl-$PERL_VERSION.tar.gz && \
```

Comment:
I had several `RUN` instructions in the beginning, and this is more convenient for modifying it indeed, but:

* This isn’t supposed to be modified too often. And when it happens, it is a good idea to update the Ubuntu packages at the same time.
* More layers also mean more disk usage. Alternatively, we could use cache mount as in https://github.com/metabrainz/musicbrainz-server/pull/3211/commits/690677d9c387d54d59ea267cd84d7cafd2a1f7ab but it might be even worse in terms of disk usage (as downloaded packages would stay on disk for a long time for nothing).